internal/sabnzbd/model/model.go

```diff
@@ -166,6 +166,10 @@ func (q *QueueStats) UnmarshalJSON(data []byte) error {
 
 // latestStat gets the most recent date's value from a map of dates to values
 func latestStat(m map[string]int) (string, int) {
+	if len(m) == 0 {
+        // Return a zero-value or suitable default
```

Comment:
The indentation here is what's causing the linter to barf. if you just run `go fmt` it should fix it up.